setup.py

```diff
@@ -17,7 +17,7 @@
 
 
 tests_require = [
-    "redis==2.4.9",
+    "redis>=2.4.9",
```

Comment:
But 3.0 doesn't exist yet and such a precaution hasn't been taken on any of the other dependencies.